charts/index.yaml

```diff
@@ -2,7 +2,15 @@ apiVersion: v1
 entries:
   rclone-copy:
   - apiVersion: v1
-    created: 2020-09-15T16:23:59.6324708+02:00
+    created: 2020-10-26T09:49:59.9562577+01:00
+    description: A helm chart for rclone copy on Kubernetes.
+    digest: 92e2565243c7084d9a03e955cba3328106ebe39419e1f19549239ecac6d87ea3
+    name: rclone-copy
+    urls:
+    - rclone-copy-1.0.0.tgz
+    version: 1.0.0
+  - apiVersion: v1
```

Comment:
This looks kinda duplicated. Can you remove index.yaml and create it from scratch again?